src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/INuGetSourcesService.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Configuration;
+
```

Comment:
`#nullable enable` here would let callers know that the methods below never accept or return null.